packages/keyring-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING**: `getAccounts` return type changed from `Promise<string>` to `string` ([#4182](https://github.com/MetaMask/core/pull/4182))
```

Comment:
We could consider preserving `async` on `getAccounts`, to avoid this breaking change. But I am OK with this changing, it's very easy to accommodate this change.

Reply:
Good point, it's very easy to accommodate but we'll have to refactor a lot of code already when releasing the complete set of atomic/exclusive operations on clients. Since it's functionally equivalent I'd say that we can come back at this later and remove the `async`.

I reverted the `async` removal part in [14ab92e](https://github.com/MetaMask/core/pull/4182/commits/14ab92e6c44552697c643e05a1bf35d0c6c4205b)

Reply:
Hmm, I'm thinking that if we intend to release https://github.com/MetaMask/core/pull/4192 on clients along with these changes, then we'll have to release https://github.com/MetaMask/core/pull/4199 too - which includes a bunch of breaking changes already, so including this one too should not change much in terms of update complexity since we'll have to release a major version already